### PR TITLE
Update writing.md

### PR DESCRIPTION
**Pull request by sanavalta**
Cleaned up the writing docs a little bit.

### CODE REVIEW

File: docs/content/en/writing.md [hunk 1]
```diff
@@ -367,7 +366,7 @@ Will be transformed into:
 }
 ```
 
-We internally add a `text` key with the markdown body that will be used for [searching](/fetching#searchfield-value) or [extending](http://localhost:3000/advanced#contentfilebeforeinsert) it.
```
Comment (benjamincanac) [inline docs/content/en/writing.md:370]: Why localhost here?

Comment (sanavalta) [reply docs/content/en/writing.md:370]: Don't ask me. That's why I deleted it.

**Comment by benjamincanac** [comment]
Hey @sanavalta,

Thank you for the improvements!

I wrote a comment on a link but besides that, it works for me!

File: docs/content/en/writing.md (hunk 1 above)
Comment (sanavalta) [reply docs/content/en/writing.md:370]: The localhost was already there in the current docs. ;)

Comment (benjamincanac) [reply docs/content/en/writing.md:370]: Oh yeah you removed it, my bad!

**Pull request merged**